commerce_canadapost.module

```diff
@@ -171,6 +172,45 @@ function commerce_canadapost_commerce_shipment_form_submit($form, FormStateInter
   $shipment->set('tracking_code', $submitted_tracking_pin);
 }
 
+/**
+ * Form validate for the commerce_store entity form.
+ */
+function commerce_canadapost_form_commerce_store_form_validate($form, FormStateInterface $form_state) {
+  $form_values = $form_state->getValues();
+
+  // If the store-wide settings are not checked, load all shipping methods that
+  // are available to that store and if there's any without settings, throw a
+  // form error.
+  if ($form_values['store_settings']) {
```

Comment:
**CHANGE REQUEST**

@shabananavas We need to be namespacing our form elements. Any contrib module can add `store_settings` in the form's submitted values, it's way too generic. As seen here, putting them in a fieldset won't change that, the form value will still be `store_settings`. It's a bit cumbersome but we need to use the prefix in both the fieldset and the form element i.e. `$form['commerce_canadapost_api']['commerce_canadapost_store_settings']` or better `$form['canadapost']['canadapost_settings']` to make it shorter, it's fairly safe.

Same with the rest e.g. `$form['canadapost']['canadapost_customer_number']`.